src/components/LastTransaction/Knob.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components/macro'
+
+export default function Knob({ type, value }) {
```

Comment:
What does `Knob` do? This name is not clear and doesn't describe what this component does.

Reply:
Rubbing salt into open sores ;-). How long I needed to find a "good" name for this component. I though about a "Ansteckknopf" like for a Jacket or something.
But maybe Pin would be better? Struggled to find a real good name for this one. Any suggestions?
This component contains a number of the last transaction of grow OR share OR spend. LastTransaction is already taken ;-)

Reply:
How about `LastTransactionAccountBalance`?

Reply:
Hm.. names are hard.. I toss `BalanceBadge` in the ring.

Reply:
Wow, thanks Martin, looks like a lot of work!
How about ``` LastTransactionValueIndicator``` or maybe only ```ValueIndicator```?

Reply:
To make everyone happy I`ve choosen: `LastTransactionValueBadge`.

Reply:
And how longer I think about I think it is a very very good name for the component. Thanks for the help.